source/model/securityLinkFilter.ts

```diff
@@ -0,0 +1,10 @@
+import { BasicPropertiesFilter } from '@flexicore/flexicore-client';
+
+export interface SecurityLinkFilter  {
```

Comment:
should extend PaginationFilter , accesses is an array of Ioperation.Access enum , not a string